configure.ac

```diff
@@ -636,6 +636,35 @@
         AC_MSG_RESULT(no)
     fi
 
+  #systemd
+    AC_ARG_WITH(systemd_includes,
```

Comment:
I think the logic here should be modified to be conditional on the check for `sd-daemon.h`and avoid modifying `CPPFLAGS` unless that header file is located.